tasks.py

```diff
@@ -731,15 +731,23 @@ def help_task(context):
 @task(
     help={
         "version": "Version of Nautobot Dev Example App to generate the release notes for.",
+        "date": "Date of the release (default: today).",
+        "keep": "Keep existing release notes files (default: False).",
```

Comment:
Is this just here for testing this command?

```suggestion
        "keep": "Keep existing release notes files. Useful for testing. (default: False).",
```

Reply:
The existing command would prompt whether to keep the notes or not. This just turns the cli message into a input.